config/default.yml

```diff
@@ -3,6 +3,7 @@ Minitest:
   Include:
     - '**/test/**/*'
     - '**/*_test.rb'
+    - '**/test/**/*_spec.rb' # For Minitest::Spec
```

Comment:
I'm confused why this isn't already covered by the first entry,`'**/test/**/*'` ?

Reply:
Oops! This is a mistake. I'm going to close this PR.